Lib/fontTools/ttLib/tables/_n_a_m_e.py

```diff
@@ -1208,3 +1208,16 @@ def visit(visitor, obj):
     if obj.version == 1:
         visitor.seen.update(obj.paletteLabels)
         visitor.seen.update(obj.paletteEntryLabels)
+
+
+@NameRecordVisitor.register(ttLib.TTFont)
+def visit(visitor, font, *args, **kwargs):
+    if hasattr(visitor, "font"):
+        return False
+
+    visitor.font = font
+    for tag in ("GSUB", "GPOS", "fvar", "CPAL", "STAT"):
```

Comment:
how about storing this list of table tags that can contain nameID references as a class attribute of NameRecordVisitor?

Reply:
Okay.

Reply:
Done.